azure-pipelines.yml

```diff
@@ -122,12 +122,12 @@ jobs:
       architecture: $(PYTHON_ARCH)
    # as noted by numba project, currently need
    # specific VC install for Python 2.7
-   # NOTE: had some issues splitting powershell
-   # command into bits and / or using condition
-   # directive, so squeezing operation to a single
-   # line for now
-  - powershell: if ($env:PYTHON_VERSION -eq 2.7) {$wc = New-Object net.webclient; $wc.Downloadfile("https://download.microsoft.com/download/7/9/6/796EF2E4-801B-4FC4-AB28-B59FBF6D907B/VCForPython27.msi", "VCForPython27.msi"); Start-Process "VCForPython27.msi" /qn -Wait}
+  - powershell: |
+      $wc = New-Object net.webclient
+      $wc.Downloadfile("https://download.microsoft.com/download/7/9/6/796EF2E4-801B-4FC4-AB28-B59FBF6D907B/VCForPython27.msi", "VCForPython27.msi")
```

Comment:
I think these long lines can also be split in `*.yml` files, see `.travis.yml`, lines 28-31. Don't know that it is worth worrying about.